packages/trader/src/App/Components/Elements/Modals/ServicesErrorModal/__tests__/account-verification-pending-modal.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import AccountVerificationPendingdModal from '../account-verification-pending-modal';
+
+describe('<AccountVerificationPendingdModal />', () => {
+    let modal_root_el: HTMLDivElement;
```

Comment:
Could you please explain the use of this?

Reply:
The component is a modal and we need a parent element to render it